core/domain/fs_domain.py

```diff
@@ -230,11 +320,28 @@ def exploration_id(self):
         return self._exploration_id
 
     def isfile(self, filepath):
-        """Checks if a file exists."""
+        """Checks if a file exists.
+
+        Args:
+            filepath: str. The path to the relevant file within the exploration.
+
+        Returns:
+            bool: True if file exists and False if it doesn't.
+        """
         return os.path.isfile(os.path.join(self._root, filepath))
 
     def get(self, filepath, version=None, mode='r'):  # pylint: disable=unused-argument
-        """Returns a bytestring with the file content, but no metadata."""
+        """Returns a bytestring with the file content, but no metadata.
+
+        Args:
+            filepath: str. The path to the relevant file within the exploration.
+            version: int or None. Optional version number of the file.
```

Comment:
No need to say "optional". Just say: "The version number of the file ..." and explain what None indicates, too.